clang/test/dpct/kernel_without_name.cu

```diff
@@ -277,3 +277,18 @@ void run_foo7(T *a, const T *b, const T *c, const T *d, const T *e, const int f,
     foo_kernel6<<<grid, block, 0, stream>>>(a, b, c, d, e, f, g, h);
   }
 }
+
+#ifndef BUILD_TEST
+template <typename T> struct kernel_type_t {
+  using Type = T;
+};
+
+template <typename T> __global__ void foo_kernel7() {
+  //CHECK:/*
+  //CHECK-NEXT:DPCT1124:{{[0-9]+}}: This type is used in other places. You may need adjust the type definition location.
```

Comment:
other places ==> call path from paralell_for sycl kernel call to current function ...